nx/include/switch/services/miiimg.h

```diff
@@ -5,7 +5,7 @@
  * @copyright libnx Authors
  */
 #pragma once
-#include "mii.h"
+#include "../services/mii.h"
```

Comment:
Add the types.h / service.h includes before this.

Reply:
Done

Reply:
Done